components/ca-context.js

```diff
@@ -0,0 +1,71 @@
+define(['document-register-element'], () => {
+    /**
+     * Web component context
+     */
+    class Context extends HTMLElement {
+
+        /** @returns {string} attribute id */
+        get id() {
+            return this.getAttribute('id');
+        }
+
+        /** @param {string} value, assigned to id attrinute */
+        set id(value) {
+            this.setAttribute('id', value);
+        }
+
+        /** @returns {string} attribute name */
+        get name() {
+            return this.getAttribute('name');
+        }
+
+        /** @param {string} value, assigned to name attrinute */
```

Comment:
nit picking especially coming from me with my funky spelling but attribute is spelt wrong.